dkist/net/attr_walker.py

```diff
@@ -74,6 +74,10 @@ def _(wlk, attr, params):
         return params.update({'hasAllStokes': True})
     if attr.value.lower() == "intensity":
         return params.update({'hasAllStokes': False})
+    if attr.value.lower() == "spectral_axis":
+        return params.update({'hasSpectralAxis': True})
+    if attr.value.lower() == "temporal_axis":
+        return params.update({'hasTemporalAxis': True})
```

Comment:
This needs some thought (probably on my part). For a start I am not sure if you can AND two physobs together, where as the union of these two keys makes sense. Also I need to check the VSO data model to see if there is a neater way of doing this.